packages/layout/src/components/cards/actuary/actuaryMachine.tsx

```diff
@@ -1,5 +1,6 @@
 import { Machine, assign } from 'xstate';
-import { ActuaryWithContactsProps } from './context';
+//import { ActuaryWithContactsProps } from './context';
```

Comment:
If this code is no longer needed it can be removed. We have git history in case we need to go back to it.

Reply:
update: commented code removed